web/vtctld2/src/app/schema/schema.component.ts

```diff
@@ -40,15 +40,15 @@ export class SchemaComponent implements OnInit {
       this.keyspaces = keyspaceNames.map(keyspaceName => {
         return {label: keyspaceName, value: keyspaceName};
       });
-      this.keyspaces.sort(this.cmp);
+      this.keyspaces.sort(SchemaComponent.cmp);
       if (this.keyspaces.length > 0 ) {
         this.selectedKeyspace = this.keyspaces[0].value;
         this.getShards(this.selectedKeyspace);
       }
     });
   }
 
-  cmp(a, b): number {
+  static cmp(a, b): number {
```

Comment:
Was there a particular motivation behind this PR?  The changes are fine, I have no problems with them.  However I noticed we have a duplicate function here https://github.com/vitessio/vitess/blob/master/web/vtctld2/src/app/dashboard/dashboard.component.ts#L90.

So for consistency should we also update that one?  Obviously code duplication is never ideal, so perhaps this could be moved to a utility class?  That could be out of scope, but I'm not sure what the scope is.